README.md

```diff
@@ -34,6 +34,24 @@ Additional notes for Mac OS X:
 
 In our [honk-models](https://github.com/honk-models) repository, there are several pre-trained models for Caffe2 (ONNX) and PyTorch. The `fetch_data.sh` script fetches these models and extracts them to the `model` directory. You may specify which model and backend to use in the config file's `model_path` and `backend`, respectively. Specifically, `backend` can be either `caffe2` or `pytorch`, depending on what format `model_path` is in. Note that, in order to run our ONNX models, the packages `onnx` and `onnx_caffe2` must be present on your system; these are absent in requirements.txt.
 
+### Raspberry Pi (RPi) Infrastructure Setup
+Unfortunately, getting the libraries to work on the RPi, especially librosa, isn't as straightforward as running a few commands. We outline our process, which may or may not work for you.
+1. Obtain an RPi, preferably an RPi 3 Model B running Raspbian.
```

Comment:
Add these instructions are tested on Raspbian Stretch. Specifically we used the image from September 8th, 2017: https://downloads.raspberrypi.org/raspbian/images/raspbian-2017-09-08/.

Reply:
Ah, okay.